### ISSUE TITLE
ACCESSIBILITY - Menu Burger Stack lacking discernible name

### ISSUE THREAD

**Issue by cpeartree** [cpeartree/Christopher_Portfolio_2#11]
**NAME:** 
Burger Stack Markup

**STATUS:** 
Assigned (Christopher Poirier)

**DESCRIPTION:** 
The

**STEPS TO REPRODUCE:** 
- Visit www.cpeartree.com
- Right click and view page source
- locate line 31 
`<a href="#" class="js-menu-open"><i class="fa-solid fa-bars"></i></a>
<br class="Apple-interchange-newline"><a href="[#](http://127.0.0.1:5500/about.html#)" class="js-menu-open"><i class="fa-solid fa-bars"></i></a>`

**EXPECTED RESULTS:**
- Appropriate element names improve SEO & Accessibility

**CONFIGURATIO:**
- This issue exists in the source code, and will be viewable on all browser/system configurations

**SEVERITY:**
- Medium

**PRIORITY:**
- Medium

**TYPE:** 
- Bug

**ASSIGNEE:**
- Christopher Poirier
